tools/cdk-build-tools/bin/cdk-build.ts

```diff
@@ -29,6 +29,11 @@ async function main() {
       desc: 'execute gen script',
       default: true,
     })
+    .option('fix', {
+      type: 'boolean',
+      desc: 'run eslint with --fix',
```

Comment:
not just eslint. pkglint and all other linters, right?

Reply:
Done